packages/filebrowser-extension/package.json

```diff
@@ -34,7 +34,6 @@
     "@jupyterlab/apputils": "^0.19.1-alpha.0",
     "@jupyterlab/coreutils": "^2.2.1-alpha.0",
     "@jupyterlab/docmanager": "^0.19.1-alpha.0",
-    "@jupyterlab/docregistry": "^0.19.1-alpha.0",
```

Comment:
did this sneak in accidentally?

Reply:
I fixed this too in https://github.com/jupyterlab/jupyterlab/pull/5436. You're right it's unrelated.